docs/tests.md

```diff
@@ -2,7 +2,7 @@
 
 Tests are written in the Go programming language to take advantage of an open source Go library called [Terratest](https://github.com/gruntwork-io/terratest).
 
-The tests also use test helper functions from [ibmcloud-terratest-wrapper](https://github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper).
+The tests also use test helper functions from [ibmcloud-terratest-wrapper](https://pkg.go.dev/github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper) ([source](https://github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper)).
```

Comment:
```suggestion
The tests also use test helper functions from the [ibmcloud-terratest-wrapper](https://github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper) module. For more information about the functions, see the Go package at [pkg.go.dev](https://pkg.go.dev/github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper)
```

how about this? trying to keep the links and link text consistent with the rest of the docs.